.github/workflows/main.yml

```diff
@@ -0,0 +1,49 @@
+name: ci
+on: [push, pull_request]
+
+jobs:
+  build_and_deploy:
+    env:
+        # latest non-preview version at the time of writing this CI script
+        BASE_VERSION: 5.0.0.2012
+        DEFAULT_REF_TO_NUGET_PUSH: refs/heads/5.0.0
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Download Nuget.exe
+      uses: NuGet/setup-nuget@v1.0.5
+      with:
+        nuget-version: 5.4.0
+    - name: setup-msbuild
+      uses: microsoft/setup-msbuild@v1
+    - name: Restore & Build
+      run: |
+        cd Xamarin.Forms.Platform.GTK
```

Comment:
does this step really need the `cd` or cannot you just use relative paths in the 2 subsequent commands?